zinc-compile/src/main/scala/sbt/inc/Doc.scala

```diff
@@ -104,6 +104,29 @@ object Doc {
 
   trait JavaDoc {
 
+    @deprecated("Use variant that takes VirtualFiles", "1.4.0")
+    /** @throws JavadocGenerationFailed when generating javadoc fails */
```

Comment:
```
[error] /home/travis/build/sbt/zinc/zinc-compile/src/main/scala/sbt/inc/Doc.scala:108:5: discarding unmoored doc comment
[error]     /** @throws JavadocGenerationFailed when generating javadoc fails */
[error]     ^
[info] No documentation generated with unsuccessful compiler run
[error] one error found
```

Reply:
Whoops. Scaladoc strings have to come above `@deprecated`.